packages/api/src/external/fhir/shared/extensions/extension.ts

```diff
@@ -2,6 +2,7 @@ import { BASE_EXTENSION_URL } from "./base-extension";
 
 // TODO #712: create this extension
 export const DATA_SOURCE_EXTENSION_URL = `${BASE_EXTENSION_URL}/data-source.json`;
+export const OPERATION_OUTCOME_EXTENSION_URL = `${BASE_EXTENSION_URL}/CodeSystem/operation-outcome`;
```

Comment:
nit: I don't think we need `CodeSystem` there, base already states those are `StructureDefinition`s.